exercises/10-exercise.py

```diff
@@ -38,27 +38,13 @@ def __call__(self, index):
 
     def get_file_list(self, datum_kwarg_gen):
         #This method is optional. It is not needed for access, but for export.
-        return ['{name}_{index}.npy'.format(name=self._name, **kwargs['index']) #**kwargs
+        return ['{name}_{index}.npy'.format(name=self._name, index =kwargs['datum_kwargs']['index']) #**kwargs
```

Comment:
I think here `datum_kwarg_gen` should generate a list of `datum_kwargs`.

For example, from datums, they're obtained by:
```py
datum_kwarg_gen = [datum['datum_kwargs'] for datum in datum_gen]
```